eve/main.yml

```diff
@@ -2671,3 +2686,42 @@ stages:
           name: Trigger multiple-nodes step with snapshot enabled
           stage_names:
             - multiple-nodes
+
+  publish-images:
+    worker: *build_worker
+    steps:
+      - ShellCommand: *wait_for_docker
+      - ShellCommand: *image_registry_login
+      - Git: *git_pull
+      - SetPropertyFromCommand:
+          name: Set shell-ui version as a property
+          property: shell_ui_version
+          command: |
+            . ./VERSION
+            echo "$VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH$VERSION_SUFFIX"
```

Comment:
Isn't this already available as `prop:metalk8s_version`?

Reply:
This property rely on the `product.txt` from pre-merge artifacts which may not be there depending from where this stage is triggered.
Anyway, I'll change this to achieve https://github.com/scality/metalk8s/pull/3406#discussion_r642814463